pkg/addons/data/ConfigMap/openshift-metrics/prometheus.yaml

```diff
@@ -247,7 +249,8 @@ data:
 
     # Scrape config for the router
     - job_name: 'openshift-router'
-      scheme: https
+      #TODO: Fix me upstream (http->https)
```

Comment:
open an issue here, refer to https://github.com/openshift/origin/issues/20375

Reply:
https://github.com/jim-minter/azure-helm/issues/101